server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/UploadEndpoint.feature

```diff
@@ -23,6 +23,11 @@ Feature: An upload endpoint should be available to upload contents
     When "username@domain.tld" upload a too big content
     Then the user should receive a request entity too large response
 
+  Scenario: Stoping an upload should work
+    Given "username@domain.tld" is starting uploading a content
+    When the user disconnect
```

Comment:
It's fixed at the last commit of https://github.com/linagora/james-project/pull/552. Please tale a look at